lang/sensei-certificates.pot

```diff
@@ -35,58 +35,72 @@ msgid "https://automattic.com"
 msgstr ""
 
 #: admin/post-types/certificate_templates.php:91
+#: build/sensei-certificates/admin/post-types/certificate_templates.php:91
```

Comment:
Is it supposed to include the `build` folder locations here? That seems odd.

Reply:
Argh, that tried to get me before on some other plugins. I had a stray `build` directory kicking around. Good catch! https://github.com/woocommerce/sensei-certificates/pull/249/commits/06f3898f01c738702c5227981ae6244fa0f67966